docs/tutorials/github-keys.md

```diff
@@ -6,16 +6,16 @@ sidebar_label: GitHub Keys
 
 ## Why we need GitHub Keys?
 
-Let's first defined GitHub as a browser-based online control system of distributed version for those software developers using the Git revision control system. Git uses SSH public key authentication and not a password authetication.  Each user of the system has to generate a key. 
+Let's first define GitHub as a browser-based online control system of distributed version for those software developers using the Git revision control system. Git uses SSH public key authetication and not a password authetication.  Each user of the system has to generate a key. 
```

Comment:
Sorry if I don't explained clearly last time, the word is not authetication is authentication, and the paragraph, this word appear two times.

>Git uses SSH public key **authetication** and not a password **authetication**